assets/src/edit-story/components/checklistCard/styles.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { focusableOutlineCSS } from '../../../design-system/theme/helpers';
+import { GRID_VARIANT } from './constants';
+import { getGridTemplateAreas } from './helpers';
+
+export const Wrapper = styled.div`
+  width: 272px;
+  display: flex;
+  border-radius: 4px;
+  background: ${({ theme }) => theme.colors.bg.secondary};
+  &:hover {
+    background: ${({ theme }) => theme.colors.interactiveBg.secondaryHover};
```

Comment:
Going to ask design about this tomorrow.  Not super sure how they want this one to go. The CTA's bg is either invisible on hover or by default depending on if secondary or tertiary button type is used. 

![disappearing button bg](https://user-images.githubusercontent.com/10720454/122308062-73fdfd00-cec0-11eb-92c5-32ee20118e0e.gif)

Reply:
(ditching hover per slack convo with ivo and juan)